README.md

```diff
@@ -222,6 +222,26 @@ TODO(awong): Figure out sauce labs integrations. Not all e2e tests should always
 run on all browsers. That's wasteful. How do we determine what should be run on
 multiple browsers as opposed to on PhantomJS in Jenkins?
 
+#### E2E Troubleshooting
+##### Is `selenium` Installed?
```

Comment:
Since selenium is part of the package, I don't think we'll run into a case where it's not installed, just issues running it. I think the `java -jar <selenium-jar-path>` test might be worth including here.